src/main/java/org/labellum/mc/dynamictreestfc/TFCSeasonManager.java

```diff
@@ -0,0 +1,78 @@
+package org.labellum.mc.dynamictreestfc;
+
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+
+import com.ferreusveritas.dynamictrees.api.seasons.ISeasonManager;
+import net.dries007.tfc.util.calendar.CalendarTFC;
+import net.dries007.tfc.util.calendar.Month;
+import net.dries007.tfc.util.climate.ClimateTFC;
+
+public class TFCSeasonManager implements ISeasonManager
+{
+    public static final TFCSeasonManager INSTANCE = new TFCSeasonManager();
+
+    @Override
+    public void updateTick(World world, long worldTicks)
+    {
+
+    }
+
+    @Override
+    public void flushMappings()
+    {
+
+    }
+
+    @Override
+    public float getGrowthFactor(World world, BlockPos rootPos, float offset)
+    {
+        return getMonthModifier(CalendarTFC.CALENDAR_TIME.getMonthOfYear());
+    }
+
+    @Override
+    public float getSeedDropFactor(World world, BlockPos rootPos, float offset)
+    {
+        return getMonthModifier(CalendarTFC.CALENDAR_TIME.getMonthOfYear()) + 0.05F;
+    }
+
+    @Override
+    public float getFruitProductionFactor(World world, BlockPos rootPos, float offset)
+    {
+        float mod = getMonthModifier(CalendarTFC.CALENDAR_TIME.getMonthOfYear());
+        return mod < 0.25F ? 0 : mod; // we don't want to fruit at all if it's the coldest part of the years
```

Comment:
Except Oranges, Apples, and Olives fruiting in November per TFC Master Data. Probably not important as we're not doing DTTFC Fruit trees yet. November is Late Fall so maybe ok?

Reply:
I think we will use TFC's handling for this stuff when we get to that rather than this